frontend/app/routes/protected/person-case/create-request.tsx

```diff
@@ -0,0 +1,159 @@
+
+import { useTranslation } from 'react-i18next';
+import * as v from 'valibot';
+
+import type { Info, Route } from './+types/create-request';
+
+import { requireAuth } from '~/.server/utils/auth-utils';
+import { i18nRedirect } from '~/.server/utils/route-utils';
+import { requestScenarioSchema } from '~/.server/validation/request-scenario-schema';
+import { sinRequestSchema } from '~/.server/validation/sin-request-schema';
+import { Button } from '~/components/button';
+import { ButtonLink } from '~/components/button-link';
+import { ErrorSummary } from '~/components/error-summary';
+import { InputRadios } from '~/components/input-radios';
+import { InputSelect } from '~/components/input-select';
+import { PageTitle } from '~/components/page-title';
+import { getFixedT } from '~/i18n-config.server';
+import { handle as parentHandle } from '~/routes/protected/layout';
+import { getLanguage } from '~/utils/i18n-utils';
+
+export const handle = {
+  i18nNamespace: [...parentHandle.i18nNamespace, 'protected'],
+} as const satisfies RouteHandle;
+
+export async function loader({ context, request }: Route.LoaderArgs) {
+  requireAuth(context.session, new URL(request.url), ['user']);
+  const t = await getFixedT(request, handle.i18nNamespace);
+  return {
+    documentTitle: t('protected:create-request.page-title'),
+    defaultFormValues: {
+      requestScenario: context.session.inPersonSINCase?.requestScenario,
+      sinRequest: context.session.inPersonSINCase?.sinRequest,
+    },
+  };
+}
+
+export function meta({ data }: Route.MetaArgs) {
+  return [{ title: data.documentTitle }];
+}
+
+export async function action({ context, request }: Route.ActionArgs) {
+  requireAuth(context.session, new URL(request.url), ['user']);
+  const lang = getLanguage(request);
+
+  const formData = await request.formData();
+  const input = {
+    requestScenario: formData.get('requestScenario') as string,
+    sinRequest: formData.get('sinRequest') as string,
+  };
+
+  const schema = v.object({ requestScenario: requestScenarioSchema(), sinRequest: sinRequestSchema() });
+  const parsedDataResult = v.safeParse(schema, input, { lang });
+
+  if (!parsedDataResult.success) {
+    return data({ errors: v.flatten<typeof schema>(parsedDataResult.issues).nested }, { status: 400 });
+  }
+
+  // If the request is valid, store it in the session and redirect to the next page
+  context.session.inPersonSINCase = {
+    ...(context.session.inPersonSINCase ?? {}),
+    ...input,
+  };
+  return i18nRedirect('routes/protected/person-case/first-name.tsx', request);
+}
+
+export default function CreateRequest({ loaderData, actionData, params }: Route.ComponentProps) {
+  const { t } = useTranslation(handle.i18nNamespace);
+  const fetcher = useFetcher<Info['actionData']>();
+  const isSubmitting = fetcher.state !== 'idle';
+  const errors = fetcher.data?.errors;
+  const baseClassName = 'py-1';
+  const inputClassName = 'w-5 h-5';
```

Comment:
No need to override the default radio style. I think you're trying to match the mockups but we already defined our radios style.